include/mender-storage.h

```diff
@@ -89,6 +89,12 @@ mender_err_t mender_storage_save_update_state(mender_update_state_t state, const
  */
 mender_err_t mender_storage_get_update_state(mender_update_state_t *state, char **artifact_type);
 
+/**
+ * @brief Delete update module state
+ * @see mender_storage_delete_update_state()
```

Comment:
> @see mender_storage_delete_update_state()

What is the purpose of this?

Reply:
Oh, I didn't even see that, I copied the comment from save_update_state without reading